test/test.jl

```diff
@@ -395,3 +395,9 @@ end
 end
 @test_throws ErrorException @testset "$(error())" begin
 end
+
+io = IOBuffer()
+@test (print(io, Base.Test.Error(:test_error, "woot", 5, backtrace())); 1) == 1
```

Comment:
See line 79 and 80 - it's a failing test that gets run in a separate process. Could record the output of that process.